.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  'env': {
+    'browser': true,
+    'node': true
+  },
+  'globals': {
+    'window': true,
+    'document': true,
+    'jQuery': true,
+    '$': true,
+    'angular': true
```

Comment:
I honestly don't care so much what framework we use. I more so don't want two overlapping frameworks. I think we should discuss what framework we may be using on future projects and use that. Personally I'd love to learn react, but I'm not sure it does everything that jQuery and Angular covers.